x-pack/plugins/ml/public/application/services/annotations_service.test.tsx

```diff
@@ -7,20 +7,24 @@
 import mockAnnotations from '../components/annotations/annotations_table/__mocks__/mock_annotations.json';
 
 import { Annotation } from '../../../common/types/annotations';
-import { annotation$, annotationsRefresh$, annotationsRefreshed } from './annotations_service';
-
+import {
+  annotationsRefresh$,
+  annotationsRefreshed,
+  AnnotationUpdatesService,
+} from './annotations_service';
 describe('annotations_service', () => {
-  test('annotation$', () => {
+  test('annotationUpdatesService', () => {
+    const annotationUpdatesService = new AnnotationUpdatesService();
     const subscriber = jest.fn();
 
-    annotation$.subscribe(subscriber);
+    annotationUpdatesService.get().subscribe(subscriber);
```

Comment:
you should test `updates$` instead

Reply:
Updated here https://github.com/elastic/kibana/pull/83216/commits/0cf7ad04d7cb582f445be2eca121a0f5946ab03e